webapp/packages/plugin-data-viewer-request-query-viewer/src/DataViewerRequestQueryViewerDialog.tsx

```diff
@@ -0,0 +1,82 @@
+ * you may not use this file except in compliance with the License.
+ */
+
+import { observer } from 'mobx-react-lite';
+
+import {
+  Button,
+  CommonDialogBody,
+  CommonDialogFooter,
+  CommonDialogHeader,
+  CommonDialogWrapper,
+  Fill,
+  useResource,
+  useTranslate,
+} from '@cloudbeaver/core-blocks';
+import type { DialogComponent } from '@cloudbeaver/core-dialogs';
+import { useService } from '@cloudbeaver/core-di';
+import { useCodemirrorExtensions } from '@cloudbeaver/plugin-codemirror6';
+import { LocalStorageSqlDataSource } from '@cloudbeaver/plugin-sql-editor';
+import { SQLCodeEditorLoader, useSqlDialectExtension } from '@cloudbeaver/plugin-sql-editor-new';
+import { ConnectionDialectResource, ConnectionInfoResource, type IConnectionInfoParams } from '@cloudbeaver/core-connections';
+import { NavNodeManagerService, NodeManagerUtils } from '@cloudbeaver/core-navigation-tree';
+import { SqlEditorNavigatorService } from '@cloudbeaver/plugin-sql-editor-navigation-tab';
+
+interface IPayload {
+  query: string;
+  connectionKey: IConnectionInfoParams;
+}
+
+export const DataViewerRequestQueryViewerDialog: DialogComponent<IPayload> = observer(function DataViewerRequestQueryViewerDialog(props) {
+  const sqlEditorNavigatorService = useService(SqlEditorNavigatorService);
+  const navNodeManagerService = useService(NavNodeManagerService);
+  const connectionInfoResource = useService(ConnectionInfoResource);
+
+  const translate = useTranslate();
+  const connectionDialectResource = useResource(DataViewerRequestQueryViewerDialog, ConnectionDialectResource, props.payload.connectionKey);
+  const sqlDialect = useSqlDialectExtension(connectionDialectResource.data);
+  const extensions = useCodemirrorExtensions();
+
+  extensions.set(...sqlDialect);
+
+  function openSqlEditor() {
+    const nodeId = NodeManagerUtils.connectionIdToConnectionNodeId(props.payload.connectionKey.connectionId);
+    const container = navNodeManagerService.getNodeContainerInfo(nodeId);
+    const connection = connectionInfoResource.get(props.payload.connectionKey);
+    const name = connection?.name ? '<' + connection.name + '> ' : 'SQL';
+
+    sqlEditorNavigatorService.openNewEditor({
+      name,
+      dataSourceKey: LocalStorageSqlDataSource.key,
+      connectionKey: props.payload.connectionKey,
+      catalogId: container.catalogId,
+      schemaId: container.schemaId,
+      query: props.payload.query,
+    });
+
+    props.rejectDialog();
+  }
+
+  return (
+    <CommonDialogWrapper size="large" fixedWidth>
+      <CommonDialogHeader title="SQL" onReject={props.rejectDialog} />
+      <CommonDialogBody noBodyPadding noOverflow>
+        <SQLCodeEditorLoader value={props.payload.query} extensions={extensions} readonly />
+      </CommonDialogBody>
+      <CommonDialogFooter>
+        <Button variant="secondary" onClick={() => props.rejectDialog()}>
+          {translate('ui_cancel')}
+        </Button>
+        <Fill />
```

Comment:
Do we want to keep using those components? I guess with tailwind we can just apply classes on the parent